GenderPayGap.WebUI/Scripts/Application/LiveSearch.js

```diff
@@ -202,6 +197,65 @@
     LiveSearch.prototype.collapseFilters = function collapseFilters() {
         $(".govuk-option-select.js-collapsible .js-container-head[aria-expanded=true]").click();
     };
+    
+    LiveSearch.prototype.sendFilterEventToGA = function sendFilterEventToGA() {
+        let selectedFilters = this.state.filter(
+            parameter =>  this.GATrackFilters.filters.map(
+                filter => filter.Group
+            ).includes(parameter.name)

```

Comment:
We can use [some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) instead.
```
this.GATrackFilters.filters.some(filter => filter.Group === parameter.name)
```

Reply:
That's neat, I didn't know about this! Thanks :)